src/content/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle.mdx

```diff
@@ -41,9 +37,10 @@ By connecting performance to user experience and interactions to performance, th
 
 To use browser session traces feature, make sure you meet these requirements:
 
+* Requires the [Pro or Pro+SPA agent type](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/#agent-types)
```

Comment:
This is Pro+SPA without spaces, but in other docs in this PR it's Pro + SPA with spaces. Consistency?